Yoast/javascript
# LINGO-467 Edit Hungarian transition words

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Edits typos and add more transition words for Hungarian.
* [Yoast SEO Free] Improves the transition words assessment for Hungarian.

## Relevant technical choices:


## Test instructions

This PR can be tested by following these steps:

* Test in Yoast SEO Free
* Set your website to Hungarian (locale: hu_HU)
* Paste a text of at least 300 words 
* Add single transition words until the transition words assessment reads: `Well done!` and the sentences containing TW are highlighted:
`semmint`
`vajon`
`ugye`
* Delete some transition words occurrences until the bullet turns orange.
* Add multiple transition words until the transition words assessment reads: `Well done!` and the sentences containing TW are highlighted:
`olybá tűnik`
`egyszer s mindenkorra`
`akkor, amikor`
* Delete some transition words occurrences until the bullet turns orange.
* Add two parts transition words to until the transition words assessment reads: `Well done!` and the sentences containing TW are highlighted:
`nem úgy..., mint...`
`oly módon..., hogy...`
`inkább..., semhogy...`
* Delete some transition words occurrences until the bullet turns orange.
* Add a three-part transition word and check that the sentence containing it is highlighted. 
`nemcsak..., hanem...is...`


## Internal test instructions:

* Run yarn test and make sure all tests pass

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-467
